mmv1/third_party/terraform/tests/resource_compute_network_endpoints_test.go.erb

```diff
@@ -260,14 +261,27 @@ func testAccCheckComputeNetworkEndpointsWithPortsDestroyed(t *testing.T, negId s
 	}
 }
 
-func testAccComputeNetworkEndpointsListEndpointPorts(t *testing.T, negId string) (map[string]struct{}, error) {
+func testAccCheckComputeNetworkEndpointsWithAllEndpointsDestroyed(t *testing.T, negId string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		endpoints, err := testAccComputeNetworkEndpointsListEndpoints(t, negId)
+		if err != nil {
+			return fmt.Errorf("unable to confirm all endpoints were destroyed: %v", err)
+		}
+		if len(endpoints) > 0 {
+			return fmt.Errorf("Not all network endpoints were deleted: %v", endpoints)
+		}
+		return nil
+	}
+}
+
+func testAccComputeNetworkEndpointsListEndpoints(t *testing.T, negId string) ([]interface{}, error) {
 	config := GoogleProviderConfig(t)
 
-	url := fmt.Sprintf("https://www.googleapis.com/compute/<% if version == 'ga' %>v1<% else %>beta<% end %>/%s/listNetworkEndpoints", negId)
+	url := fmt.Sprintf("https://www.googleapis.com/compute/v1/%s/listNetworkEndpoints", negId)
```

Comment:
Just to double-check, will this also return endpoints created with the beta API?

Reply:
not sure how that happened, I only copied and pasted it... thanks for catching!